dlt/common/storages/configuration.py

```diff
@@ -324,3 +326,119 @@ def make_file_url(local_path: str) -> str:
         netloc is never set. UNC paths are represented as file://host/path
         """
         return make_fsspec_url("file", local_path, None)
+
+
+@dataclass
+class WithLocalFiles(BaseConfiguration):
+    """Mixin to BaseConfiguration that shifts relative locations into `local_dir` and allows for a few special locations.
+    :pipeline: in the pipeline working folder
```

Comment:
I know this was already like this, but why not "pipeline_working_dir"? This way it is clear what this variable means.